scripts/globals/mobskills/evisceration.lua

```diff
@@ -0,0 +1,31 @@
+---------------------------------------------
+-- Evisceration
+-- Delivers a fivefold attack
+-- Type: Physical
+-- Range: Melee
+---------------------------------------------
+require("scripts/globals/mobskills")
+require("scripts/settings/main")
```

Comment:
It doesn't look like any settings are used in this file, so this require isn't needed.